apps/website/app/page.tsx

```diff
@@ -8,10 +8,32 @@ import {
   CardTitle,
 } from "@repo/ui/components/ui/card";
 import { ArrowBigDownDash, CircleGauge } from "lucide-react";
+import path from 'path';
+import fs from 'fs';
+import matter from 'gray-matter';
 
 const inter = Inter({ subsets: ["latin"] });
 
-export default function Home() {
+async function getLatestBlogs() {
+  const blogDirectory = path.join(process.cwd(), '/blogs');
+  const files = fs.readdirSync(blogDirectory);
+
+  return files
+    .map((filename) => {
+      const filePath = path.join(blogDirectory, filename);
+      const fileContent = fs.readFileSync(filePath, 'utf-8');
+      const { data } = matter(fileContent);
+
+      return {
+        slug: filename.replace('.md', ''),
+        ...data,
+      };
+    })
+    .sort((a, b) => new Date(b.date) - new Date(a.date)) // Sort by date
+    .slice(0, 3); // Get the first 3 blogs
+}
+export default async function Home() {
+  const blogs = await getLatestBlogs();
   return (
     <div className={`min-h-screen bg-neutral-light ${inter.className}`}>
       <header className="flex flex-col items-center justify-between space-y-4 px-6 py-4 md:flex-row md:space-y-0">
```

Comment:
Let's move the header/nav to the root layout.tsx, that way the /blog and blog posts will inherit it.

We can also move the `bg-neutral-light` and any other classes that should wrap the body